earth2mip/inference_medium_range.py

```diff
@@ -155,15 +155,17 @@ def process(initial_time):
             # select first history level
             verification_torch = verification_torch[:, -1]
             for metric in metrics:
-                outputs = metric.call(verification_torch, data)
+                indexes = [i for i,name in enumerate(model.out_channel_names) if name in model.in_channel_names]
+                outputs = metric.call(verification_torch,data[:,indexes,:,:]
+                #outputs = metric.call(verification_torch, data)
                 for name, tensor in zip(metric.output_names, outputs):
                     v = tensor.cpu().numpy()
-                    for c_idx in range(len(model.out_channel_names)):
```

Comment:
I believe out_channels is correct here since we are operating on `v` which is output data.

Rather we should attempt to fix the problem up at the datasource part where it errors.